app/javascript/tests/patient/Viewers.test.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import Viewers from '../../components/patient/Viewers';
+import { mockPatient1, mockPatient2, mockPatient3 } from '../mocks/mockPatients';
+
+function getWrapper(mockPatient, viewers) {
+  return shallow(<Viewers patient_id={mockPatient.id} viewers={viewers} />);
+}
+
+describe('Viewers', () => {
+  const wrapper1 = getWrapper(mockPatient1, []);
```

Comment:
Sure; although I actually copied this from https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/tests/patient/PatientPage.test.js - so it may be worth revisiting how this is done across the front end tests to make sure we're consistent

Reply:
That's actually the one file that has it that way and I can fix it in my latest PR - the only thing with that one is the same wrappers are used in different tests, but here they're only used once in each test